openstack/backup-replication/templates/replication-configmap.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: swift-http-import
+  labels:
+    component: backup-replication
+
+data:
+{{- range $source_region, $source := .Values.source }}
+  {{ $source_region }}.yaml: |
+    swift:
+      auth_url: {{ $.Values.target.auth_url }}
+      user_name: {{ $.Values.target.user_name }}
+      user_domain_name: {{ $.Values.target.user_domain_name }}
+      project_name: {{ $.Values.target.project_name }}
+      project_domain_name: {{ $.Values.target.project_domain_name }}
+      password: {{ $.Values.target.password }}
+    statsd:
+      hostname: {{ $.Values.statsd.hostname }}
```

Comment:
I would suggest to hardcode to hostname to `statsd` completly and remove it from the value file. Also the service name is derived from `$.Values.statsd.hostname`